k8s/pkg/apis/longhorn/v1beta2/engine.go

```diff
@@ -138,6 +138,11 @@ type EngineSpec struct {
 	UnmapMarkSnapChainRemovedEnabled bool `json:"unmapMarkSnapChainRemovedEnabled"`
 	// +optional
 	Active bool `json:"active"`
+	// +optional
+	SnapshotMaxCount int `json:"snapshotMaxCount"`
+	// +kubebuilder:validation:Type=string
+	// +optional
+	SnapshotMaxSize int64 `json:"snapshotMaxSize,string"`
```

Comment:
Why `kubebuilder:validation:Type=string` instead of `int`?

Reply:
I followed `VolumeSpec.Size`. @derekbit, could you share the idea to use string validation? This was added from https://github.com/longhorn/longhorn-manager/pull/1136. Thank you.